docs/recipes/event-delegation.md

```diff
@@ -0,0 +1,45 @@
+# Event delegation 
+
+In the 1st example we use `most.fromEvent` to create a stream from
```

Comment:
In general, I think we should link to the docs for the first occurrence of every API function mentioned in the recipe.  What do you think?